package.json

```diff
@@ -110,10 +110,10 @@
     "ipfs-block": "~0.7.1",
     "ipfs-block-service": "~0.14.0",
     "ipfs-multipart": "~0.1.0",
-    "ipfs-repo": "~0.20.0",
+    "ipfs-repo": "^0.22.1",
```

Comment:
We should keep the `~` for packages below `1.0.0`, as stated in the [js-ipfs guidelines](https://github.com/ipfs/community/blob/master/js-code-guidelines.md#dependency-versions).

Reply:
good catch! :)